bin/install.sh

```diff
@@ -108,6 +102,8 @@ sudo find /usr/share/locale -mindepth 1 -maxdepth 1 ! -name 'en*' ! -name 'de*'
 
 cd ~/screenly && git rev-parse HEAD > ~/.screenly/latest_screenly_sha
 
+echo -e "Screenly version: $(git rev-parse --abbrev-ref HEAD)@$(git rev-parse HEAD)\n$(cat /etc/os-release | grep 'PRETTY_NAME')" > ~/version.md
```

Comment:
The latter part will likely fail in a dockerized version. Also, `lsb_release` is probably a lot cleaner than `cat`-ing the file.